README.md

```diff
@@ -1,5 +1,57 @@
 # stream-freshdesk
 
+A data stream generator for the Freshdesk REST API, written in python 3.
+
+## Quick start
+
+1. Install
+
+    Clone this repository, and then install using setup.py. We recommend using a virtualenv:
+
+    ```bash
```

Comment:
actually, you can just do `pip install tap-freshdesk` instead of these three lines